### PR TITLE
Optimize trigger github action rules

### PR DESCRIPTION
**Pull request by lbbniu**
#### What this PR does / why we need it:
Optimize build images trigger github action rules
#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by lbbniu** [comment]
see:
https://github.com/lbbniu/crane/pull/11
https://github.com/lbbniu/crane/pull/14

**Review by zsnmwy — COMMENTED** [review]
/LGTM

File: .github/workflows/preview.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
     types: [opened, synchronize, reopened, closed]
     paths:
       - 'docs/**'
-      - '*.md'
+      - '**.md'
```
Comment (qmhu) [inline .github/workflows/preview.yml:9]: Just curious, what's the difference between *.MD and **.MD here?

Comment (zsnmwy) [reply .github/workflows/preview.yml:9]: *.md  => Matches all .md files at the **root** of the repository.
**.md  => Matches all .md files in the repository.

image

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths

**Pull request merged**